evap/static/css/evap.css

```diff
@@ -474,3 +474,7 @@ li.menu-indent {
 .table-striped tbody tr.sortable-ghost {
     background: #E5E5E5;
 }
+
+select {
+    width: 100%;
+}
```

Comment:
we also do on other pages, but there they are already sized correctly. i can think about something else if need be.